tests/10-issue.js

```diff
@@ -26,95 +26,325 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
-          this.test.cell = {columnId: issuerName, rowId: this.test.title};
-          should.exist(issuerResponse);
-          should.not.exist(err);
-          issuerResponse.status.should.equal(201);
-          should.exist(issuedVc, `Expected VC from ${issuerName} to exist.`);
-          testCredential({credential: issuedVc});
-        });
-        // ensure that issued StatusList Credential contain correct properties
-        it('MUST have correct properties when dereferencing ' +
-          '"credentialStatus.statusListCredential"', async function() {
-          this.test.cell = {columnId: issuerName, rowId: this.test.title};
-          const {slc} = await getSlc({issuedVc});
-          testSlCredential({slCredential: slc});
+        describe('BitstringStatusList Entry', function() {
+          it('Any expression of the data model in this section ' +
+            'MUST be expressed in a conforming verifiable credential ' +
+            'as defined in [VC-DATA-MODEL-2.0].',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=Any%20expression%20of%20the%20data%20model%20in%20this%20section%20MUST%20be%20expressed%20in%20a%20conforming%20verifiable%20credential%20as%20defined%20in%20%5BVC%2DDATA%2DMODEL%2D2.0%5D.';
+            should.exist(issuerResponse,
+              'Expected an issuer response.');
+            should.not.exist(err,
+              'Expected no errors in the issuer response.');
+            issuerResponse.status.should.equal(201,
+              'Expected response http code 201.');
+            should.exist(issuedVc, `Expected VC from ${issuerName} to exist.`);
+            testCredential({credential: issuedVc});
+          });
+          it('If present, the id value is expected to be a URL ' +
+            'that identifies the status information associated ' +
+            'with the verifiable credential. ' +
+            'It MUST NOT be the URL for the status list.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20present%2C%20the%20value%20is%20expected%20to%20be%20a%20URL%20that%20identifies%20the%20status%20information%20associated%20with%20the%20verifiable%20credential.';
+            if(!!issuedVc.credentialStatus.id) {
+              // TODO test for URI
+              issuedVc.credentialStatus.id.should.be.a('string',
+                'Expected credentialStatus.id to be a string.'
+              );
+            } else {
+              this.skip();
+            }
+          });
+          it('The type property MUST be BitstringStatusListEntry. ',
+            async function() {
+              this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20type%20property%20MUST%20be%20BitstringStatusListEntry.';
+              issuedVc.credentialStatus.should.have.own.property(
+                'type').to.be.a('string',
```

Comment:
does this chain work? that's awesome if it does.

Reply:
yes, it was suggested by @BigBlueHat !